sat/cli/bootsys/platform.py

```diff
@@ -239,7 +239,7 @@ def __init__(self, host, host_keys=None):
         super().__init__()
         self.host = host
         self.host_keys = host_keys
-        self.success = False
+        self.success = True
```

Comment:
I think this needs to remain set to `False` initially in order to ensure that if the process exits unexpectedly (e.g. uncaught exception), then the process should be deemed unsuccessful.

```suggestion
        self.success = False
```